src/GDBDebugSession.ts

```
@@ -988,7 +988,9 @@ export class GDBDebugSession extends LoggingDebugSession {
         args: DebugProtocol.ContinueArguments
     ): Promise<void> {
         try {
-            await mi.sendExecContinue(this.gdb, args.threadId);
+            if(this.isRunning === false) {
```

Please don't suppress this error. If the client asks to resume, but the thread is running already the adapter should send back an error indicating that.

This is especially true since your check doesn't match the continue arguments. You are checking against the state of all threads, but may be resuming just a single thread.